bip-0353.mediawiki

```diff
@@ -26,7 +26,7 @@ Various Bitcoin and other cryptocurrency applications have developed human-reada
 
 The DNS provides a standard, global, hierarchical namespace mapping human-readable labels to records of various forms. Using DNSSEC, the DNS provides cryptographic guarantees using a straightforward PKI which follows the hierarchical nature of the DNS, allowing for stateless and even offline validation of DNS records from a single trusted root.
 
-Further, because DNS queries are generally proxied through ISP-provided or other resolvers, DNS queries usually do not directly expose the queryer's IP address. Further, because of the prevalence of open resolvers, the simplicity of the protocol, and broad availability of DNS recursive resolver implementations, finding a proxy for DNS records is trivial.
+Further, because DNS queries are generally proxied through ISP-provided or other resolvers, DNS queries usually do not directly expose the querier's IP address. Further, because of the prevalence of open resolvers, the simplicity of the protocol, and broad availability of DNS recursive resolver implementations, finding a proxy for DNS records is trivial.
```

Comment:
Per https://english.stackexchange.com/questions/214204/what-is-the-proper-way-to-say-queryer, it looks like there are many possible spellings:

- queryer
- querier
- queryist
- querist (the only variant that "passes" my browser spelling checker plugin)
- querant

I'm fine with leaving the text as it is, unless others prefer to change it.

Reply:
![image](https://github.com/user-attachments/assets/e7950c17-a4f0-4686-a324-93e6d1d015bf)

![image](https://github.com/user-attachments/assets/1f1bc24c-72a1-44c8-b902-e6f2b2f293e4)

- I also see "queryer" in https://www.wordthesauri.com/thesaurus/questioner/ for instance, so it doesn't appear to be egregiously wrong.